applications/HDF5Application/custom_io/hdf5_model_part_io.cpp

```diff
@@ -277,13 +285,13 @@ void ModelPartIO::WriteSubModelParts(ModelPart const& rModelPart)
         }
         if (GlobalNumberOfElements(*it) > 0)
         {
-            WriteContainerIds(*mpFile, sub_model_part_path + "/ElementIds", it->Elements(), info);
-            StoreWriteInfo(sub_model_part_path + "/ElementIds", info);
+            ModelPartIO current_model_part_io(mpFile, sub_model_part_path);
+            current_model_part_io.WriteElements(it->Elements());
```

Comment:
this directly works with an h5 file? I am impressed

Reply:
It works :)....